libraries/BLE/examples/central/central.ino

```diff
@@ -0,0 +1,94 @@
+
+#include "ArduinoBLE.h"
+#include "BLEAttribute.h"
+#include "BLECharacteristicImp.h"
+#include "BLEProfileManager.h"
+
+// LED pin
+#define LED_PIN   13
+
+void setup() {
+    Serial.begin(115200);
+    Serial.println("test---");
+    
+    // set LED pin to output mode
+    pinMode(LED_PIN, OUTPUT);
+
+    // begin initialization
+    BLE.begin();
+    Serial.println(BLE.address());
+    
+    BLE.startScanning("LED");

```

Comment:
If the environment is noisy, the available api is slow, need more buffer in the stack. I think we need consider those questions
1. The RAM
2. Does device need buffer the ADV's data?
3. How to let user process the RAW ADV data?
4. Filter ADV